server.js

```diff
@@ -20,8 +14,71 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+  res.send(
+    'Netflix-titles! End-points: /movies , /movies/country/:country , /movies/title/:title , /movies/releaseyear/:year, /movies/:id/  '
```

Comment:
I'm wondering where this is displayed, because on / path, I get all endpoints, not this "Netflix-titles! End-points:" text at all?